.github/workflows/config.yml

```diff
@@ -121,37 +121,46 @@ jobs:
           tar -xzf Windows.tar.gz
           & .\OpenStudio-${env:OS_VERSION}+${env:OS_SHA}-Windows\bin\openstudio.exe workflow\run_simulation.rb -x workflow\sample_files\base.xml --hourly ALL --add-component-loads --add-stochastic-schedules
 
+  merge-results:
+    runs-on: ubuntu-latest
+    needs: [run-workflow1-tests, run-workflow2-tests, run-unit-tests]
+    steps:
+      - name: Merge Artifacts
+        uses: actions/upload-artifact/merge@v4
+        with:
+          name: results
+          pattern: results-*
+          delete-merged: true
```

Comment:
The `upload-artifact` action no longer allows automatic merging of artifacts, instead they now require [using a separate action to achieve this](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#merging-multiple-artifacts).